src/card/basic/card.directive.js

```diff
@@ -3,26 +3,140 @@
  * @name patternfly.card.directive:pfCard
  * @restrict A
  * @element ANY
- * @param {headTitle=} Title for the card - required
- * @param {subTitle=} Subtitle for the card - optional
- * @param {showTopBorder=} Show Top Border, true shows top border, false (default) hides top border - optional
- *
+ * @param {string} headTitle Title for the card
+ * @param {string=} subTitle Sub-Title for the card
+ * @param {boolean=} showTopBorder Show/Hide the blue top border. True shows top border, false (default) hides top border
+ * @param {boolean=} showTitlesSeparator Show/Hide the grey line between the title and sub-title.
+ * True (default) shows the line, false hides the line
+ * @param {object=} footer footer configuration properties:<br/>
+ * <ul style='list-style-type: none'>
+ * <li>.iconClass  - (optional) the icon to show on the bottom left of the footer panel
+ * <li>.text       - (optional) the text to show on the bottom left of the footer panel, to the right of the icon
+ * <li>.href       - (optional) the href link to navigate to when the footer href is clicked
+ * <li>.callBackFn - (optional) user defined function to call when the footer href is clicked
+ * </ul>
+ * *Note: If a href link and a callBackFn are specified, the href link will be called
+ * @param {object=} filter filter configuration properties:<br/>
+ * <ul style='list-style-type: none'>
+ * <li>.filters    - drop down items for the filter.
+ *<pre class=''>
+ *  Ex:  'filters' : [{label:'Last 30 Days', value:'30'},
+ *                    {label:'Last 15 Days', value:'15'},
+ *                    {label:'Today', value:'today'}]</pre>
+ * <li>.defaultFilter - integer, 0 based index into the filters array
+ * <li>.callBackFn - user defined function to call when a filter is selected
+ * </ul>
  * @description
  * Directive for easily displaying a card with html content
  *
  * @example
- <example module="patternfly.card">
+ <example module="test">
```

Comment:
I thought it would be confusing since this ngDoc example isn't just using the one directive, but two.  For ngDoc examples, the module name can be anything, it's just for the ngDoc, it's like a third party app.   'test' probably isn't the best name either, I'll change it to 'demo' so it's clearer.